gnucash/report/standard-reports/transaction.scm

```diff
@@ -13,6 +13,9 @@
 ;; - introduced account/transaction substring/regex matcher
 ;; - add custom sorter in scheme
 ;; - common currency - optionally show original currency amount
+;;   and enable multiple data columns
+;; - add informational box, summarising options used, useful
+;;   to troubleshoot reports
```

Comment:
In general try to avoid doing things in a commit that you didn't specify in the commit message.
In this case it's pretty harmless as it's just an extra comment. I got more confused with your later commits.

If you feel like it you could take this opportunity to practice rewriting your git history using a mixture of git reset/git add -i and so on. If you do, create a new branch at this commit, from which you can the revert (mixed mode) the last commit. That will put your changes back in your working directory ready to be added/committed. Then a git add -i will allow you to only select some changes to go into a commit. Commit that and repeat until all changes are split up in separate commits. From there on you can cherry-pick or rebase the later commits from your other branch on top of this new one and continue to clean up.

While not mandadory it's certainly useful to learn these techniques as it will allow you to write better history (both for yourself as the other developers looking into your changes).